stories/index.tsx

```diff
@@ -178,6 +178,19 @@ storiesOf('Griddle main', module)
     )
   })
 
+  .add('with custom default sort', () => {
+    return (
```

Comment:
I don't think this grid needs to be pre-sorted, no. This test is derived from the [test that follows](https://github.com/NewBoCo/Griddle/blob/7ecddf1ec5ab8d933bb2fbc544788c251f691829/stories/index.tsx#L194-L206).